src/app/_utilities/getEpisodeDuration.ts

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from "react";
+import { formatEpisodeDuration } from "@/app/_utilities/formatEpisodeDuration";
+
+export function getEpisodeDuration(src) {
```

Comment:
Because this functions uses hooks (useState, useEffect), it's technically also a hook. It's common practice to call hooks useSomething, so in this case `useEpisodeDuration`

Reply:
fixed ✅